src/main/config/messages_template.xml

```diff
@@ -808,6 +808,11 @@ See the accompanying LICENSE file for applicable license.
     <reason>A content reference in a constrained document type cannot be resolved because it would violate one of the document constraints "%1".</reason>
     <response>The current constrained document may only reuse content from documents with equivalent constraints.</response>
   </message>
+  
+  <message id="DOTX077D" type="DEBUG">
+    <reason>Rewrite conref content @id from "%1" to "%2".</reason>
+    <response></response>
```

Comment:
@infotexture Do you have suggestion for better copy? I don't think we should have anything in the response, because it would be something like

> If you don't want @id to be rewritten, change the conreffed source to use some other ID value that is not already in use in the conreffing or other conreffed content.

😬

Reply:
I don't think the current message will be meaningful to most people -- maybe something like "Resolving content references in [file] results in duplicate ID [id]. Rewriting resolved version to [newid]"?

Reply:
Change the copy but the "in [file]" bit comes from log context so removed that. Now the message is e.g.

> [conref] file:/Volumes/tmp/src/conref/b.dita:7:20: [DOTX077I][INFO]: Resolving content references results in duplicate ID 'bar'. Rewriting resolved version to 'd22e17'.